COMPILING.md

```diff
@@ -0,0 +1,140 @@
+Jamulus can be compiled for Linux, Windows and macOS as follows. Note also the contributed [installation scripts](https://github.com/jamulussoftware/installscripts) for use unattended installs. There are also reports from people who successfully compile and run Jamulus on BSDs.
+
+---
+
+
+## Download sources
+
+For .tar.gz [use this link](https://github.com/jamulussoftware/jamulus/archive/latest.tar.gz) to download the latest release
+
+For .zip [use this link](https://github.com/jamulussoftware/jamulus/archive/master.zip)
+
+## Linux
+
+#### Install dependencies
+
+On Ubuntu-based distributions 18.04+, Debian 9+ or 10 and Raspberry Pi Raspbian Buster release or later:
+
+* build-essential
+* qt5-qmake
+* qtdeclarative5-dev
+* qt5-default
+* qttools5-dev-tools
+* libjack-jackd2-dev
+
+#### On Fedora
+
+* qt5-qtdeclarative-devel
+* jack-audio-connection-kit-dbus
+* libqt5concurrent5
+* jack-audio-connection-kit-devel
+
+#### For all desktop distributions
+
+[QjackCtl](https://qjackctl.sourceforge.io/) is optional, but recommended to configure JACK.
+
+
+### Standard desktop build
+
+```
+make distclean
+qmake
+make
+sudo make install
+```
+
+`make distclean` is optional but ensures a clean build environment. `make install` is optional and puts the Jamulus binary into `/usr/local/bin`.
+
+**Users of Raspberry Pi** You may want to compile the client on another machine and run the binary on the Raspberry Pi. In which case the only libraries you need to run it are those for a [headless server](Server-Linux#running-a-headless-server) build, but _with_ the JACK sound packages. 
+
+
+### “Headless” server build
+
```

Comment:
Just a random thought: Would you say that there are more people compiling headless for Linux since they're using a linux server? If that's the case, we might want to move it above the desktop build.

Reply:
Perhaps, but it would be a bit odd to assume this I think, since the people reading this will have chosen not to install binary files to use Jamulus. In the long run, I would expect this document to be read hardly ever at all :-)